### PR TITLE
Api 4722 add smart cards

### PR DESCRIPTION
**Pull request by apeirats-va**
Is this a new product?
Yes - smart-cards

Has this been deployed successfully from a branch?
Yes

Are load balancer rules being removed or renamed?
No

Are ingress definitions being removed or renamed?
No

Are Kubernetes namespaces being removed or renamed?
No

Has DevOps been consulted on resource limits?
No, refer to current DU at https://github.com/department-of-veterans-affairs/health-apis-smart-cards-deployment

### CODE REVIEW

**Comment by apeirats-va** [comment]
Found a last minute bug on target app. Getting that fixed. Will reopen this PR after app and DU is re-built, and deployer is re-tested on branch.

File: Jenkinsfile [hunk 1]
```diff
@@ -29,6 +29,7 @@ def products() {
   products["scheduling"] = ["health_apis_jenkins","shankins"]
   products["sfdc-mpi-msg-receiver"] = ["health_apis_jenkins"]
   products["sfdc-mvi-ent"] = ["health_apis_jenkins"]
+  products["smart-cards"] = ["health_apis_jenkins"]
```
Comment (jhulbert-va) [inline Jenkinsfile:32]: PGD uses `shankins`, do you want that as well? It may also be worth creating a new channel for `pteracuda` deployments.

Comment (jblefkove-va) [reply Jenkinsfile:32]: we prefer to keep spamming your channel

Comment (apeirats-va) [reply Jenkinsfile]: Added shankins for now. I'll defer to shanktopus if you guys want us out of shankins.

**Review by jblefkove-va — APPROVED** [review]

**Review by eclendenning-va — APPROVED** [review]

**Review by jhulbert-va — APPROVED** [review]

**Pull request merged**